Header header
geometry_msgs/PointStamped position
string type
float32 distance_to
int8 scout_id
bool smoothed
bool collected
